### PR TITLE
LegacyAmixerControl: fix bytes control support mapped to component type

### PR DESCRIPTION
**Pull request by mgaio**
When bytes control is mapped to component type, to/from Blackboard() calls
can not be used.

In this case, blackboard has to be accessed using blackboardWrite() and
blackboardRead()

Signed-off-by: Miguel Gaio <miguel.gaio@intel.com>

### CODE REVIEW

**Comment by dawagner** [comment]
Please give more details in the commit message: you didn't explain why to/fromBlackboard does not work.

File: legacy/LegacyAmixerControl.cpp [hunk 1]
```diff
@@ -281,39 +287,44 @@ bool LegacyAmixerControl::accessHW(bool receive, std::string &error)
...
+            // Go through all indexes
+            for (index = 0; index < elementCount; index++) {
+
+                // Read data from blackboard (beware this code is OK on Little Endian machines only)
+                value = fromBlackboard();
+
+                if (isDebugEnabled()) {
+
+                    // The "info" method has been shadowed by a local variable
+                    this->info() << "Writing alsa element " << controlName
```
Comment (dawagner) [inline legacy/LegacyAmixerControl.cpp]: With this patch, that log isn't produced in case of byte controls anymore.

Comment (mgaio) [inline legacy/LegacyAmixerControl.cpp]: yes but one bytes per line, if you have at worst 512 bytes is not really helpfull.
So just skip as done in TLV bytes control

Comment (dawagner) [inline legacy/LegacyAmixerControl.cpp]: Sure but you could come up with a smarter way to log.

**Pull request merged**